File: java/oc/src/main/java/oc/good/CommandProcessor.java
package oc.good;

import java.util.ArrayList;
import java.util.List;

public class CommandProcessor {
    public static List<Command> commands=new ArrayList<>();
    public static void process(String command){
        System.out.println("Starting Your Command");
        for (Command oCommand:
             commands) {
            if(oCommand.verb.equals(command))
            {
                oCommand.operation.doSomething(command);
            }
        }
        System.out.println("Done Your Command");
    }
}

File: python/good/CommandProcessor.py
class CommandProcessor:
    commands=[]
    @staticmethod
    def process(command):
        print("Starting Command " + command)
        for oCommand in CommandProcessor.commands:
            if oCommand.verb == command:
                oCommand.action(command)

        print("Ending Command " + command)
class Command:
    def __init__(self, verb, action):
        self.verb = verb
        self.action = action